sdk/eventhub/tests.yml

```diff
@@ -5,11 +5,6 @@ extends:
   parameters:
     MaxParallel: 6
     ServiceDirectory: eventhub
+    SDKType: client
```

Comment:
With this embedded, I think we need a `tests.data.yml` for the T1 libraries, no?

Reply:
Yes